core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.ts

```diff
@@ -113,88 +114,96 @@ export class ContributorAdminDashboardPageComponent implements OnInit {
     private contributorDashboardAdminStatsBackendApiService: ContributorDashboardAdminStatsBackendApiService,
     private userService: UserService,
     private contributorDashboardAdminBackendApiService: ContributorDashboardAdminBackendApiService,
-    private modalService: NgbModal
+    private modalService: NgbModal,
+    private alertsService: AlertsService
   ) {}
 
   ngOnInit(): void {
     this.loadingMessage = 'Loading';
     this.activeTab = this.TAB_NAME_TRANSLATION_SUBMITTER;
     this.contributorDashboardAdminStatsBackendApiService
       .fetchCommunityStats()
-      .then(response => {
-        this.translationReviewersCountByLanguage =
-          response.translation_reviewers_count;
-        this.questionReviewersCount = response.question_reviewers_count;
-
-        this.languageChoices = AppConstants.SUPPORTED_AUDIO_LANGUAGES.map(
-          languageItem => {
-            return {
-              id: languageItem.id,
-              language: this.putEnglishLanguageNameAtFront(
-                languageItem.description
-              ),
-            };
-          }
-        );
-
-        this.selectedLastActivity = 0;
-
-        this.userService.getUserInfoAsync().then(userInfo => {
-          const username = userInfo.getUsername();
-
-          if (username === null) {
-            return;
-          }
-          this.isQuestionCoordinator = userInfo.isQuestionCoordinator();
-          this.isTranslationCoordinator = userInfo.isTranslationCoordinator();
-
-          if (this.isTranslationCoordinator) {
-            this.CONTRIBUTION_TYPES.push(
-              this.TAB_NAME_TRANSLATION_SUBMITTER,
-              this.TAB_NAME_TRANSLATION_REVIEWER
-            );
+      .then(
+        response => {
+          this.translationReviewersCountByLanguage =
+            response.translation_reviewers_count;
+          this.questionReviewersCount = response.question_reviewers_count;
+
+          this.languageChoices = AppConstants.SUPPORTED_AUDIO_LANGUAGES.map(
+            languageItem => {
+              return {
+                id: languageItem.id,
+                language: this.putEnglishLanguageNameAtFront(
+                  languageItem.description
+                ),
+              };
+            }
+          );
+
+          this.selectedLastActivity = 0;
```

Comment:
TBH, don't think I know what this means ... though from the diff it looks like I might have missed it before. But how can an activity be a number?

Reply:
@seanlip , I'm not sure why this is in the diff, maybe something got moved up or down ?  There is a dropdown that allows selection of the number of days back into the past that we want to show contributions for (7,30, 90), its called lastActivity. selectedLastActivity is the number that has been selected.

Reply:
Oh I see, thanks! Maybe could we say selectedLastActivityLookbackDays or something like that? That way it's at least clear that it's a number of days.

Thanks!